docs/getting_started/installation.md

```diff
@@ -108,9 +108,13 @@ conda activate sf
 pip install -U secretflow
 ```
 
-## GPU support
+## GPUs support
 
-NVIDIA's CUDA and cuDNN are typically used to accelerate the training and testing of Tensoflow and PyTorch deep learning models. Tensoflow and PyTorch are both deep learning backends for SecretFlow, and if you want to use GPU acceleration in SecretFlow, you can follow these steps:
+NVIDIA's CUDA and cuDNN are typically used to accelerate the training and testing of Tensoflow and PyTorch deep learning models. Tensoflow and PyTorch are both deep learning backends for SecretFlow.
+
+[JAX]([google/jax: Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more (github.com)](https://github.com/google/jax)) is the frontend highly recommended by the SecretFlow team. JAX can compile and run your NumPy code on accelerators, such as GPUs and TPUs.Now we supply GPUs supports to JAX in SecretFlow.
```

Comment:
`[JAX]([google/jax: Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more (github.com)](https://github.com/google/jax))`
-> `[JAX](https://github.com/google/jax)`

Reply:
Sorry for my mistake and I simplified hyperlinks of JAX as you suggested, Thank you! @zhouaihui